vpc/vpc-vpn-bastion.yaml

```diff
@@ -295,6 +295,10 @@ Resources:
         IpProtocol: tcp
         FromPort: 5555
         ToPort: 5555
+      - CidrIp: 0.0.0.0/0
+        IpProtocol: udp
+        FromPort: 40000
+        ToPort: 40000
```

Comment:
If I get this https://github.com/SoftEtherVPN/SoftEtherVPN/blob/1e17c9bcfd7e7b31756aa5389bcbff76c2c9c88a/src/Cedar/UdpAccel.h#L139 right we have to set `ToPort` to `44999`

Reply:
You are right. I've debuged actuall issue using tcpdump.
Should I close this PR and create a new one?